glotpress.php

```diff
@@ -27,7 +27,7 @@
  */
 
 define( 'GP_VERSION', '2.1.0-alpha' );
-define( 'GP_DB_VERSION', '940' );
+define( 'GP_DB_VERSION', '950' );
```

Comment:
I wondered where the number came from.

Either just an incrementing number (I chose ten, but any increment would do) or date.

Date is kind of ugly and does limit us to once change a day :wink:.